docs/demo-weather-agent.md

```diff
@@ -0,0 +1,166 @@
+# Weather Agent Demo
+
+This document provides detailed steps for running the **Weather Agent** demo proof-of-concept (PoC).
+
+In the [Import New Agent](#import-new-agent) section, you'll build and deploy the [`acp_weather_service`](https://github.com/kagenti/agent-examples/tree/main/acp/acp_weather_service) agent.
+
+In the [Import New Tool](#import-new-tool) section, you'll build and deploy the [`weather_tool`](https://github.com/kagenti/agent-examples/tree/main/mcp/weather_tool) tool.
+
+> **Prerequisites:**  
+> Ensure you've completed the Kagenti platform setup as described in the [Installation](../cn-demos.md#installation) section. This demo uses `ACP` protocol, so you will not need `OPENAI_API_KEY` in your environment setup.
+
+You should also open the Agent Platform Demo Dashboard as instructed in the [Run the Demo](../cn-demos.md#run-the-demo) section.
```

Comment:
I think this is basic enough that you can just put the link here. It seems unnecessary to force users to go to another page just for this.